src/SignalR/clients/java/signalr/test/signalr.client.java.Tests.javaproj

```diff
@@ -6,6 +6,7 @@
     <IsTestProject>true</IsTestProject>
     <!-- Installing Java on ARM will take some work -->
     <SkipHelixArm>true</SkipHelixArm>
+    <SkipHelixQueues>(Alpine.312.Amd64.Open)Ubuntu.1604.Amd64.Open@mcr.microsoft.com/dotnet-buildtools/prereqs:alpine-3.12-helix-20200908125345-56c6673</SkipHelixQueues>
```

Comment:
Did you look into why this one failed? I assume the java bits we install will work on alpine

Reply:
```
PlatformArch: linux-x64
Downloading from: https://netcorenativeassets.blob.core.windows.net/resource-packages/external/linux/java/jdk-10.0.2_linux-x64_bin.tar.gz
Installing java from jdk-10.0.2_linux-x64_bin.tar.gz https://netcorenativeassets.blob.core.windows.net/resource-packages/external/linux/java/jdk-10.0.2_linux-x64_bin.tar.gz
Unpacking to /root/helix/work/workitem/java
/root/helix/work/workitem
+ '['  '='  ]
+ export 'JAVA_HOME=/root/helix/work/workitem/java'
+ chmod +x ./gradlew
+ ./gradlew '-Dorg.gradle.daemon=false' '-PpackageVersion=6.0.0-ci' test
./gradlew: exec: line 183: /root/helix/work/workitem/java/bin/java: not found
```

https://helixre8s23ayyeko0k025g8.blob.core.windows.net/dotnet-aspnetcore-refs-heads-halter73-15277-44d69a8221e240158c/signalr.client.java.Tests--/console.173bce24.log?sv=2019-07-07&se=2020-11-12T22%3A27%3A23Z&sr=c&sp=rl&sig=e5XON14ymqsnY%2B0Uz4Aw7tFRUCcanUqbVmZEHJfIZ7s%3D

https://dev.azure.com/dnceng/public/_build/results?buildId=864541&view=results

Why would you assume the java bits would work on alpine? We already skip this on ARM.

Reply:
Alpine is just a linux distro right? It's still x64 and should work the same as all the other linux distro's we run on.

From the error, it looks like either tar didn't work correctly, or the scripting didn't work https://github.com/dotnet/aspnetcore/blob/219ecd6880120215137f4d6ef0dd2aa62c780af2/eng/targets/Helix.targets#L4

Reply:
The thing that makes Alpine very different from most other Linux distros is that it uses musl instead of glibc as its c standard library and they are not ABI compatible.